src/views/shared/SecurityBeacon.js

```diff
@@ -45,7 +45,14 @@ define(['okta', 'util/Animations'], function (Okta, Animations) {
     }
   }
 
-  function antiPhishingMessage (image, host, shown) {
+  function antiPhishingMessage (image, host) {
+    function showQtip() {
+      if(image.is(':visible')){
+        image.qtip('show');
+      }
+    }
+    $(window).on('resize.securityBeaconQtip', _.debounce(showQtip, 300));
```

Comment:
Anyway, I have to create a function for using inside the `_.debounce`, o why not using it twice?